relay-extensions.graphql

```diff
@@ -0,0 +1,211 @@
+# THIS FILE WAS ADDED TO FIX THE GRAPHQL LSP ISSUE DEFINED HERE https://github.com/graphql/graphiql/issues/2180#issuecomment-1088713441
```

Comment:
defines all the custom relay directives that the graphql LSP fails to validate against due to missing definitions